### DISCUSSION TITLE
Transfer to org discussions

### DISCUSSION THREAD

**Discussion by sschu** [keycloak/keycloak#20106] [Q&A]
@keycloak/maintainers-external I see that warning that Team Discussions will be deprecated by a 8th of May. I would migrate it to organization discussions as suggested if you don't object by Thursday, ok?

**Comment by thomasdarimont** [comment]
Fine by me

**Comment by tnorimat** [comment]
Yes, no problem.

**Comment by sschu** [comment]
Done...

**Reply by thomasdarimont** [reply]
Thanks! Where can we find the new discussions?

**Reply by sschu** [reply]
I found it in the normal discussions in category "Team Posts". It looks like this also makes it public, not sure if there is a way to configure it differently...
